docs/adrs/rework-batch-analysis-architecture.md

```diff
@@ -0,0 +1,82 @@
+# Rework Batch Analysis Architecture
```

Comment:
Is this just for batch analysis or does it affect upload also? My understanding was the latter.

Reply:
Data import and analysis will probably use the same process.

Reply:
We can use the same process for data import if we like it and think it's a good model. During our discussions we were mainly focused on analysis just to start with (as far as I understood). Partially because it helps not to bloat scope, but also we were anticipating more input from design on import and wanted to wait until they got further in the process.